libs/components/src/locales/en-GB.ts

```diff
@@ -103,6 +104,103 @@ const enGB: Locale = {
 	},
 	splitButton: {
 		showMoreActionsLabel: 'Show more actions',
+	},
+	videoPlayer: {
+		'Audio Player': 'Audio Player',
```

Comment:
I think we can just import the locale from video.js to avoid copying it:
```suggestion
	videoPlayer: {
		videoJsLocale: <...imported from video.js...>,
```